docs:(quickstart): set installed version to v1.0.3

GlennBullingham opened the pull request:
Update kubectl deployment examples to install Mayastor v1.0.3

Signed-off-by: GlennBullingham <glenn.bullingham@datacore.com>

GlennBullingham committed:
docs:(quickstart): set installed version to v1.0.3

Update kubectl deployment examples to install Mayastor v1.0.3

Signed-off-by: GlennBullingham <glenn.bullingham@datacore.com>

anupriya0703 commented on quickstart/deploy-mayastor.md:
````diff
@@ -279,7 +279,7 @@ msp-operator-7849d59fcd-mcw5b   1/1     Running   0          21s
 {% tab title="Command \(GitHub Latest\)" %}
 ```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/master/v1.0.2/mayastor-daemonset.yaml
+kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/master/v1.0.3/mayastor-daemonset.yaml
````
The above URL throws a 404 error. 
Correct URL- https://raw.githubusercontent.com/openebs/Mayastor/v1.0.3/deploy/mayastor-daemonset.yaml

GlennBullingham replied:
Good catch, fixed.

avishnu reviewed (approved):
Apart from comment from @anupriya0703, rest look good.

GlennBullingham committed:
fix(quickstart): url for mayastor io daemonset

 URL given for mayastor-daemonset.yaml incorrectly included
 a reference to the now defunct master branch

Signed-off-by: GlennBullingham <glenn.bullingham@datacore.com>
